DPL-200: Add update message exporter

sdjmchattie opened the pull request:
Closes #471 

Changes proposed in this pull request:

* Add the exporter for update messages.
* Handle database connection issues and log if manual intervention would be required.

stevieing commented on crawler/processing/update_sample_exporter.py:
```diff
@@ -0,0 +1,250 @@
+import copy
```
Maybe a bit more top level documentation on what is going on?

sdjmchattie replied:
Yes that's entirely fair.  Will do that.

sdjmchattie commented:
@stevieing I've added documentation.  I also refactored a property away that wasn't really being used.  And I identified a bug around properties that was masked by the functionality still appearing to behave as expected.  I've fixed that as well.
